CHANGELOG.adoc

```diff
@@ -20,6 +20,8 @@
 - https://github.com/eclipse-syson/syson/issues/887[#887] [explorer] Rework the structure of standard and user libraries in the explorer:
 * Rename `StandardLibraryDirectory` to `LibrariesDirectory`.
 * Rename `ReferencedLibraryDirectory` to `UserLibrariesDirectory`.
+- https://github.com/eclipse-syson/syson/issues/906[#906] [explorer] Ensure that a non-library imported model is never moved into the _User libraries_ directory.
```

Comment:
Could you add an integration test or a cypress test please?

Reply:
done, I move the `setIsImported` call to `ElementInitializerSwitch` to cover more edition use cases (now an imported resource will be correctly flagged as non-imported if its content has been updated via the explorer of via a tool in the diagram).
I added two unit tests for `ElementInitializerSwitch` to cover these use cases.